specs/altair/light-client/full-node.md

```diff
@@ -34,6 +35,19 @@ def compute_merkle_proof_for_state(state: BeaconState,
     ...
 ```
 
+### `block_to_light_client_header`
+
+```python
+def block_to_light_client_header(block: SignedBeaconBlock) -> BeaconBlockHeader:
```

Comment:
It looks like a general method to get the beacon block header from the block. What do you think about making it a `BeaconBlock -> BeaconBlockHeader` method `beacon_block_to_header`?

Reply:
nvm I just saw the attempt of #3151!